src/ai/backend/common/types.py

```diff
@@ -1125,3 +1126,12 @@ class AcceleratorMetadata(TypedDict):
     display_unit: str
     number_format: AcceleratorNumberFormat
     display_icon: str
+
+
+# Redis database IDs depending on purposes
+REDIS_STAT_DB: Final = 0
+REDIS_RLIM_DB: Final = 1
+REDIS_LIVE_DB: Final = 2
+REDIS_IMAGE_DB: Final = 3
+REDIS_STREAM_DB: Final = 4
+REDIS_STREAM_LOCK: Final = 5
```

Comment:
Now that we have these DB definitions on a `common` package please remove duplicate declarations at `ai.backend.manager.defs` and migrate all usages referencing old constants under `.defs`  to these newer ones.